cmd/nerdctl/compose.go

```diff
@@ -82,135 +69,44 @@ func newComposeCommand() *cobra.Command {
 }
 
 func getComposer(cmd *cobra.Command, client *containerd.Client, globalOptions *types.GlobalCommandOptions) (*composer.Composer, error) {
```

Comment:
After the below suggestion, this `getComposer` becomes a wraper of 2 function calls: `getComposeOptions` in `cmd` side and `GetComposer` in `pkg` side.

Although it seems redundant, I think it's fine for now to keep it to minimize the change. Otherwise, you need to change every `getCompoer` call in compose commands to first call `getComposeOptions` and then call `compose.GetComposer`.